src/Misc/layoutbin/runsvc.sh

```diff
@@ -10,10 +10,11 @@ if [ -f ".path" ]; then
     echo ".path=${PATH}"
 fi
 
-# insert anything to setup env when running as a service
+nodever=${GITHUB_ACTIONS_RUNNER_FORCED_NODE_VERSION:-node16}
```

Comment:
`GITHUB_ACTIONS_RUNNER` seems to be the prefix we use most often for these kind of env vars